runtime/src/asset.rs

```diff
@@ -2428,7 +2501,9 @@ mod tests {
                 token.name.clone(),
                 token.name.clone(),
                 token.total_supply,
-                true
+                true,
+                AssetType::default(),
+                Identifiers::default(),
             ));
 
```

Comment:
Need a test case/line that verifies that `AssetType` and `Identifiers` have been added properly.

Reply:
I added a couple of `assert_eq`s after that.